evm/rlp/blocks.py

```diff
@@ -1,3 +1,4 @@
+from typing import Type  # noqa: F401
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically